src/components/fx/hover.js

```diff
@@ -155,7 +155,7 @@ exports.loneHover = function loneHover(hoverItem, opts) {
 
 // The actual implementation is here:
 function _hover(gd, evt, subplot, noHoverEvent) {
-    if((subplot === 'pie' || subplot === 'sankey') && !noHoverEvent) {
+    if(subplot === 'sankey' && !noHoverEvent) {
```

Comment:
I couldn't see any purpose to having pie call `Fx.hover` other than throttling - which really shouldn't be an issue for `pie`, so I removed it from this special handling. Am I missing anything? Should we do the same for `sankey` and 🔪 this block entirely?

Reply:
Looks like `sankey` does `Fx.hover` [here](https://github.com/plotly/plotly.js/blob/9bfbabfba9a5975edb0b6f49c625f6f7db7acd4b/src/traces/sankey/plot.js#L134) and [here](https://github.com/plotly/plotly.js/blob/9bfbabfba9a5975edb0b6f49c625f6f7db7acd4b/src/traces/sankey/plot.js#L191), but if you feel adventurous, sure, go ahead and patch `sankey/plot.js` to :hocho: this block.

Reply:
simplified sankey -> https://github.com/plotly/plotly.js/pull/2117/commits/e08e37ae58296d0abcedb6f02f21f9153bb6676b
Also: I notice that the way we had it was only throttling *event emission*, not the actual item selection which is normally the slow part. Again, throttling shouldn't be an issue for either of these trace types, this is just more confirmation of that.

Reply:
https://github.com/plotly/plotly.js/commit/e08e37ae58296d0abcedb6f02f21f9153bb6676b is very nice. :ok_hand: